### PR TITLE
Example config file and version change

### PR DESCRIPTION
**Pull request by berggren**
* Add example configuration
* Change version
* Some config checks
* Add missing dependencies

Reviewer: @onager

### CODE REVIEW

File: timesketch.conf [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Timesketch configuration
+#
+# The default location for this configuration file in /etc/timesketch.conf
```
Comment (Onager) [inline timesketch.conf]: * is

File: timesketch.conf [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# If you put it somewhere else you can pass the path to tsctl.py
+# Example:
+#
+# $ tsctl.py -c /path/to/this/timesketch.conf runserver
+
+# Turn off debugging.
+# It is a security risk to have this enabled in production.
+DEBUG = False
+
+# This should be a unique random string. Don't share this with anyone.
```
Comment (Onager) [inline timesketch.conf]: What does this actually secure? Is this the seed for XSRF protection, or something else?

File: timesketch.conf (hunk 1 above)
Comment (berggren) [inline timesketch.conf]: Done

File: timesketch.conf (hunk 2 above)
Comment (berggren) [inline timesketch.conf]: Added more infor

**Comment by Onager** [comment]
As discussed:
*  It'd be nice to generate the secret key on install
*  It'd be nice at some point to move the test dependencies somewhere nicer

These are for later though.

LGTM

**Pull request merged**